classes/class-frontend.php

```
@@ -1,412 +1,265 @@
-	 * @param string $meta_key
-	 * @param boolean $single
-	 * @return void
+	 * @param mixed  $metadata  Existing metadata (null = not intercepted).
+	 * @param int    $object_id Post ID.
+	 * @param string $meta_key  Meta key.
+	 * @param bool   $single    Whether to return a single value.
+	 * @return mixed
 	 */
-	public function filter_post_meta( $metadata = null, $object_id, $meta_key, $single ) {
-		if ( true === lsx_currencies()->convert_to_single && 'price' === $meta_key ) {
+	public function filter_post_meta( $metadata, $object_id, $meta_key, $single ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$single'. (undefined)

(UnusedFormalParameter)